src/hooks/screen/useChatInfoHook.ts

```diff
@@ -28,7 +27,7 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
   } = useGroupInfo();
   const {localDb} = useLocalDatabaseHook();
   const [loadingChannelInfo, setLoadingChannelInfo] = React.useState<boolean>(false);
-  const {selectedChannel, goBack} = useChatUtilsHook();
+  const {isLoadingFetchingChannelDetail, selectedChannel, goBack} = useChatUtilsHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `isLoadingFetchingChannelDetail` state variable from `useChatUtilsHook()` seems to be a good change. It will allow the component to react to the loading state of fetching channel details, which can be used to show or hide loading indicators in the UI.

```diff
-  const {selectedChannel, goBack} = useChatUtilsHook();
+  const {isLoadingFetchingChannelDetail, selectedChannel, goBack} = useChatUtilsHook();
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->